msobooking.yaml

```diff
@@ -0,0 +1,436 @@
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      ol.projectedenddatetime:
+        type: "ol.projectedenddatetime"
+        column: "RELDBDATE"
+    name: "temporaryrelease"
+
+  releaseofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "RELOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "RELOFF"
+    name: "releaseofficer"
+
+  dnatest:
+    fqn: "ol.medicaltest"
+    entitySetName: "MSODnatest"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "DNA_SAMPLE_DATE"
+      ol.datetimeadministered:
+        type: "ol.datetimeadministered"
+        column: "DNA_SAMPLE_DATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      ol.status:
+        type: "ol.status"
+        column: "DNA_SAMPLE_STATUS"
+    name: "dnatest"
+
+  sentence:
+    fqn: "ol.sentence"
+    entitySetName: "MSOSentence"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCE"
+      ol.notes:
+        type: "ol.notes"
+        column: "SENTENCE"
+    name: "sentence"
+
+  officer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "IDOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "IDOFF"
+    name: "officer"
+
+  dnaofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "DNA_SAMPLE_OFFICER"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "DNA_SAMPLE_OFFICER"
+    name: "dnaofficer"
+
+  committingofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "COMOFF"
+    name: "committingofficer"
+
+  committingcounty:
+    fqn: "ol.location"
+    entitySetName: "MSOCommittingcounty"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "COUNTY"
+      ol.county:
+        type: "ol.county"
+        column: "COUNTY"
+    name: "committingcounty"
+
+  releaselocation:
+    fqn: "ol.location"
+    entitySetName: "MSOReleaselocation"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "RELLOC"
+      location.name:
+        type: "location.name"
+        column: "RELLOC"
+      general.notes:
+        type: "general.notes"
+        column: "RELADDR1"
+    name: "releaselocation"
+
+  immigrationstatus:
+    fqn: "ol.immigrationstatus"
+    entitySetName: "MSOImmigrationstatus"
```

Comment:
I just looked in the table at the distinct values in INS for this and they are: 
US CITIZEN

 909999VA8
 85768MC0

Easy enough to put a condition on the entity set so only those with the first value is read in.

Reply:
There are only 68 values of US CITIZEN - and  it seems unclear whether the random strings have any meaning or not. The blanks are already removed by the script that is creating the cleaned table - To remove these two values I would just delete those particular values in that particular column - but I am wondering if it makes more sense to do that, to ask about these values, or to just not integrate this column at all (only 70 values / 146,350 without much meaning since it is only affirmative)

Reply:
Either ask Sonya about this and comment it out, or delete the two random values. Also these distinct values implies a bunch of blank but not null cells, if not filtered out would create empty looking entities in this entity set.